src/test/java/reactor/netty/tcp/TcpServerTests.java

```diff
@@ -934,6 +936,77 @@ public void testGracefulShutdown() throws Exception {
 		            .verifyComplete();
 	}
 
+	@Test(expected = ChannelBindException.class)
+	public void testTcpServerWithDomainSocketsNIOTransport() {
+		LoopResources loop = LoopResources.create("testTcpServerWithDomainSocketsNIOTransport");
+		try {
+			TcpServer.create()
+			         .runOn(loop, false)
+			         .bindAddress(() -> new DomainSocketAddress("/tmp/test.sock"))
+			         .bindNow();
+		}
+		finally {
+			loop.disposeLater()
+			    .block(Duration.ofSeconds(30));
+		}
+	}
+
+	@Test(expected = IllegalArgumentException.class)
+	public void testTcpServerWithDomainSocketsWithHost() {
+		TcpServer.create()
+		         .bindAddress(() -> new DomainSocketAddress("/tmp/test.sock"))
+		         .host("localhost")
+		         .bindNow();
+	}
+
+	@Test(expected = IllegalArgumentException.class)
+	public void testTcpServerWithDomainSocketsWithPort() {
+		TcpServer.create()
+		         .bindAddress(() -> new DomainSocketAddress("/tmp/test.sock"))
+		         .port(1234)
+		         .bindNow();
+	}
+
+	@Test
+	public void testTcpServerWithDomainSockets() throws Exception {
+		if (!LoopResources.hasNativeSupport()) {
```

Comment:
Seems like a could candidate for JUnit `assumeTrue()`

Reply:
changed